src/lib/AppRegistry.tsx

```diff
@@ -51,6 +51,7 @@ import renderWithLoadProgress from "./utils/renderWithLoadProgress"
 import { Schema, screenTrack as track } from "./utils/track"
 
 YellowBox.ignoreWarnings([
+  "Warning: RelayResponseNormalizer: Payload did not contain a value for field `id: id`. Check that you are parsing with the same query that was used to fetch the payload.",
```

Comment:
this change is unrelated to this PR, but this warning is not helpful so let's get rid of it.